package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "node-vibrant",
-  "version": "3.1.5",
+  "name": "@tailwind/node-vibrant",
+  "version": "3.2.0",
```

Comment:
I think this is a major version change since we're bumping major versions of node and typescript.

Reply:
Or since we just forked this for the first time, do we make this version 1?

Reply:
I'd rather keep the versions "in-sync", but yeah I'll bump up the major version.